test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -35,13 +35,10 @@ describe('33acrossBidAdapter:', function () {
           ttx: {
             w: 1024,
             h: 728,
-            pxr: 2,
             vp: {
               w: 800,
               h: 600
-            },
-            ah: 500,
-            mtp: 0
+            }
```

Comment:
I wanted to put this comment by lines 482-483, but Github wouldn't let me:

I'd recommend setting clientWidth & clientHeight to something other than 800 & 600 in your test fixture because 800 & 600 are also the values you're using for innerWidth & innerHeight in your test fixture. This will allow you to feel confident that your tests are validating that innerWidth & innerHeight are taking priority over clientWidth & clientHeight.